src/modules/land_detector/land_detector_params_mc.c

```diff
@@ -31,6 +31,21 @@
  *
  ****************************************************************************/
 
+/**
+ * Multicopter land detection trigger time
+ *
+ * Time it takes to go through all three lnad detection phases
```

Comment:
```suggestion
 * Time it takes to go through all three land detection phases
```

Reply:
Maybe also worth mentioning the three phases here?

Reply:
Makes sense. Changed by interactive rebase:
https://github.com/PX4/PX4-Autopilot/pull/17440/commits/3f2c631fb42e56e2ffa00db1bb80cc0f02c9c4ba#diff-420a85704f4369bcc9ebb06b8081fcc86b1d8795340f4a845a47eda3c502c498R37-R39